manifest.xml

```diff
@@ -15,5 +15,7 @@
   <depend package="base/orogen/types"/>
   <depend package="tools/logger"/>
   <depend package="orogen"/>
+  <depend package="soup" />
+  <depend package="seasocks"/>
```

Comment:
These will have to be defined in the rock package set. Please create the corresponding PR.